instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/TracingRequestHandler.java

```diff
@@ -31,21 +31,27 @@ final class TracingRequestHandler extends RequestHandler2 {
       ContextKey.named(TracingRequestHandler.class.getName() + ".Timer");
   private static final ContextKey<Boolean> REQUEST_SPAN_SUPPRESSED_KEY =
       ContextKey.named(TracingRequestHandler.class.getName() + ".RequestSpanSuppressed");
+  private static final String SEND_MESSAGE_REQUEST =
+      "com.amazonaws.services.sqs.model.SendMessageRequest";
+  private static final String DYNAMODBV2 = "com.amazonaws.services.dynamodbv2.model.";
```

Comment:
slight nitpick: perhaps the usage of these variables provides enough context, but it might be a little more intuitive if the names indicated that they reference the names of classes

```suggestion
  private static final String SEND_MESSAGE_REQUEST_CLASS =
      "com.amazonaws.services.sqs.model.SendMessageRequest";
  private static final String DYNAMODBV2_CLASS = "com.amazonaws.services.dynamodbv2.model.";
```